assemblies/pentaho-war/pom.xml

```diff
@@ -25,7 +25,7 @@
     <tika-core.version>1.15</tika-core.version>
     <xmlgraphics-commons.version>2.2</xmlgraphics-commons.version>
     <c3p0.version>0.9.1.2</c3p0.version>
-    <xml-apis.version>2.0.2</xml-apis.version>
+    <slf4j.version>1.7.7</slf4j.version>
```

Comment:
I think this needs to be rebased. Changing slf4j is not part of the Xerxes upgrade.

Reply:
done